app/services/solidus_klarna_payments/place_order_with_customer_token_service.rb

```diff
@@ -38,18 +38,12 @@ def customer_token
     end
 
     def fetch_customer_token
-      @customer_token_response = Klarna
-                                 .client(:payment)
-                                 .customer_token(
-                                   payment_source.authorization_token,
-                                   customer_token_params
-                                 )
-
-      customer_token = begin
-        @customer_token_response.token_id
-      rescue NoMethodError
-        nil
-      end
+      customer_token = SolidusKlarnaPayments::CreateCustomerTokenService
```

Comment:
it's not clear if this is a fetch or a create. Also, why not using the same pattern used in other places to allow class configuration (eg. https://github.com/solidusio-contrib/solidus_klarna_payments/blob/e30439f683565838231367f4f099bd651286a46e/lib/solidus_klarna_payments/configuration.rb#L8)?

Reply:
This is just a service that convert the authorizationToken in a customerToken, if you have a better name for this service I'll be happy to change it.

I considered customizing it, but differently from the other 2 services we don't have to provide our own service, but just to re-use it outside of the `PlaceOrderWithCustomerTokenService`, so at the end, it would have added any benefits, but just over-engineering.

Reply:
I don't have a better name but the method that wraps this call is named `fetch_customer_token` so it is strange to see:

```
def fetch_customer_token
  SolidusKlarnaPayments::CreateCustomerTokenService
end
```

maybe we should change the method name as well?

Reply:
Yes, probably would be better to rename the method into `generate_and_store_customer_token`

Reply:
Done!